cmd/podman/images/push.go

```diff
@@ -100,6 +103,7 @@ func pushFlags(cmd *cobra.Command) {
 	_ = cmd.RegisterFlagCompletionFunc(formatFlagName, common.AutocompleteManifestFormat)
 
 	flags.BoolVarP(&pushOptions.Quiet, "quiet", "q", false, "Suppress output information when pushing images")
+	flags.BoolVar(&pushOptions.Rm, "rm", false, "remove the manifest list if push succeeds")
```

Comment:
I think this should really be a different PR (and discussion).  I have mixed feelings about it as I think users should really be using `podman manifest remove`.   Users may ask to remove normal images at some point and then want `--rm-force` etc.  I prefer to draw the lines early to not open too many doors for complexity.

Reply:
Docker doesn't have a similar option. I think I lean towards @vrothberg's thoughts here.